src/libmongoc/tests/json-test.c

```diff
@@ -317,9 +320,6 @@ process_sdam_test_hello_responses (bson_t *phase,
                                             generation);
          bson_mutex_unlock (&topology->mutex);
       }
-   } else {
-      test_error (
-         "test does not have 'responses' or 'applicationErrors' array");
```

Comment:
The new [discover_load_balancer](https://github.com/mongodb/specifications/blob/289f4053106261bae80ba098c8afeb885b26acc2/source/server-discovery-and-monitoring/tests/load-balanced/discover_load_balancer.yml) SDAM test does not include the `responses` field. I think this is intentional since no hello responses need to be processed to transition to a `LoadBalanced` topology.